beetsplug/web/__init__.py

```diff
@@ -25,6 +25,11 @@
 from werkzeug.routing import BaseConverter, PathConverter
 import os
 import json
+import tempfile
+from string import Template
```

Comment:
This should go with the other `from anything_but_spam import spam` imports, instead of the middle of the `import eggs` ones.